packages/ui/v2/core/Dialog.tsx

```diff
@@ -78,7 +79,8 @@ export const DialogContent = React.forwardRef<HTMLDivElement, DialogContentProps
               : props.size == "md"
               ? "p-8 sm:max-w-[48rem]"
               : "p-8 sm:max-w-[35rem]",
-            "overflow-y-auto overscroll-auto md:h-auto md:max-h-[inherit]",
+            "max-h-[560px] overscroll-auto md:h-auto md:max-h-[inherit]",
```

Comment:
max-height is required in case the content in any dialog is too big for screen. 

<img width="477" alt="Screenshot 2022-12-29 at 10 50 02 PM" src="https://user-images.githubusercontent.com/53316345/209987312-22ad9290-e0ef-492d-8a9d-2d4107a1d687.png">